http-client/src/main/java/no/nav/familie/http/config/NaisProxyCustomizer.kt

```diff
@@ -14,11 +18,18 @@ import org.springframework.web.client.RestTemplate
 interface INaisProxyCustomizer : RestTemplateCustomizer
 
 @Component
-class NaisProxyCustomizer : INaisProxyCustomizer {
+class NaisProxyCustomizer(@Value("\${familie.nais.proxy.connectTimeout:15000}") val connectTimeout: Int,
+                          @Value("\${familie.nais.proxy.socketTimeout:15000}") val socketTimeout: Int,
+                          @Value("\${familie.nais.proxy.requestTimeout:15000}") val requestTimeout: Int) : INaisProxyCustomizer {
```

Comment:
Fint om alle har en idé om at dette er fornuftige verdier for våre repos - og evt. overstyrer der det er spesialbehov med ekstremt lang svartid (gjelder socket og request)